hezar/preprocessors/preprocessor.py

```diff
@@ -49,13 +50,7 @@ def push_to_hub(
         raise NotImplementedError
 
     @classmethod
-    def load(
```

Comment:
@HamedBabaei jan, such core methods like `load`, `save`, `push_to_hub` function defenitions might not exceed the 120 characters limit, but for readability, we prefer to wrap them. Is there any solution to keep such cases untouched?

Reply:
After turning `isort` backend to `black`, the current method keeps such cases untouched. I will try to find them and roll them back as much as I could do!